js/mexc.js

```diff
@@ -2094,4 +2095,66 @@ module.exports = class mexc extends Exchange {
             throw new ExchangeError (feedback);
         }
     }
+
+    async fetchFundingRateHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        //
+        // Gets a history of funding rates with their timestamps
+        //  (param) symbol: Future currency pair
+        //  (param) limit: not used by mexc
+        //  (param) since: not used by mexc
+        //  (param) params: Object containing more params for the request
+        //  return: [{symbol, fundingRate, timestamp, dateTime}]
+        //
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchFundingRateHistory() requires a symbol argument');
+        }
```

Comment:
you should also have a check like this for limit, because `mexc` requires it

Reply:
@samgermain looks like it works without `page_size` and without `page_num`:

- https://contract.mexc.com/api/v1/contract/funding_rate/history?symbol=BTC_USDT

But not without a symbol:

- https://contract.mexc.com/api/v1/contract/funding_rate/history